pkg/proxy/ipvs/proxier.go

```diff
@@ -739,6 +739,10 @@ func (proxier *Proxier) syncProxyRules() {
 	activeIPVSServices := map[string]bool{}
 	// currentIPVSServices represent IPVS services listed from the system
 	currentIPVSServices := make(map[string]*utilipvs.VirtualServer)
+	// activeBindAddrs represents ip address successfully bind to DefaultDummyDevice in this round of sync
+	activeBindAddrs :=  map[string]bool{}
+	// currentBindAddrs represents ip addresses bind to DefaultDummyDevice from the system
+	currentBindAddrs := map[string]bool{}
```

Comment:
Seems like `currentBindAddrs` only used this character in `cleanLegacyBindAddr()`, and don't need index. So maybe `ListBindAddress()` returned list of address is enough.

Reply:
you mean there is no need to use a `map` for `currentBindAddrs`, string list is enough